packages/wallet-core/src/store/mutations.ts

```diff
@@ -131,6 +131,15 @@ export default {
 
     state.history[network]![walletId].push(transaction);
   },
+  REMOVE_NFT_FROM_STATE(
```

Comment:
This is not correctly named
All mutations do something to the state, so `FROM_STATE` should not be required